charts/argo-cd/values.yaml

```diff
@@ -315,8 +315,11 @@ configs:
     # -- Known hosts to be added to the known host list by default.
     # @default -- See [values.yaml]
     knownHosts: |
```

Comment:
There were some changes in manifest, so I followed.
ref: https://github.com/argoproj/argo-cd/blob/v2.7.4/manifests/base/config/argocd-ssh-known-hosts-cm.yaml

<img width="967" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/6555255d-2848-4261-bfa8-c26edd1eec11">